build.py

```diff
@@ -108,338 +109,400 @@ def target_platform():
-    p = subprocess.Popen(['tar', '--strip-components=1', '-xf', tarfile],
-                         cwd=targetdir)
+    log_verbose("untar {} into {}".format(tarfile, targetdir))
+    p = subprocess.Popen(["tar", "--strip-components=1", "-xf", tarfile], cwd=targetdir)
     p.wait()
-    fail_if(p.returncode != 0,
-            'untar {} into {} failed'.format(tarfile, targetdir))
+    fail_if(p.returncode != 0, "untar {} into {} failed".format(tarfile, targetdir))
 
 
-def gitclone(cwd, repo, tag, subdir):
+def gitclone(cwd, repo, tag, subdir, org):
     # If 'tag' starts with "pull/" then it must be of form
     # "pull/<pr>/head". We just clone at "main" and then fetch the
     # reference onto a new branch we name "tritonbuildref".
     if tag.startswith("pull/"):
         log_verbose('git clone of repo "{}" at ref "{}"'.format(repo, tag))
-        p = subprocess.Popen([
-            'git', 'clone', '--recursive', '--depth=1', '{}/{}.git'.format(
-                FLAGS.github_organization, repo), subdir
-        ],
-                             cwd=cwd)
+        p = subprocess.Popen(
+            [
+                "git",
+                "clone",
+                "--recursive",
+                "--depth=1",
+                "{}/{}.git".format(org, repo),
+                subdir,
+            ],
+            cwd=cwd,
+        )
         p.wait()
-        fail_if(p.returncode != 0,
-                'git clone of repo "{}" at branch "main" failed'.format(repo))
+        fail_if(
+            p.returncode != 0,
+            'git clone of repo "{}" at branch "main" failed'.format(repo),
+        )
 
         log_verbose('git fetch of ref "{}"'.format(tag))
         p = subprocess.Popen(
-            ['git', 'fetch', 'origin', '{}:tritonbuildref'.format(tag)],
-            cwd=os.path.join(cwd, subdir))
+            ["git", "fetch", "origin", "{}:tritonbuildref".format(tag)],
+            cwd=os.path.join(cwd, subdir),
+        )
         p.wait()
         fail_if(p.returncode != 0, 'git fetch of ref "{}" failed'.format(tag))
 
-        log_verbose('git checkout of tritonbuildref')
-        p = subprocess.Popen(['git', 'checkout', 'tritonbuildref'],
-                             cwd=os.path.join(cwd, subdir))
+        log_verbose("git checkout of tritonbuildref")
+        p = subprocess.Popen(
+            ["git", "checkout", "tritonbuildref"], cwd=os.path.join(cwd, subdir)
+        )
         p.wait()
-        fail_if(p.returncode != 0,
-                'git checkout of branch "tritonbuildref" failed')
+        fail_if(p.returncode != 0, 'git checkout of branch "tritonbuildref" failed')
 
     else:
         log_verbose('git clone of repo "{}" at tag "{}"'.format(repo, tag))
-        p = subprocess.Popen([
-            'git', 'clone', '--recursive', '--single-branch', '--depth=1', '-b',
-            tag, '{}/{}.git'.format(FLAGS.github_organization, repo), subdir
-        ],
-                             cwd=cwd)
+        p = subprocess.Popen(
```

Comment:
Are you formatting as described here? https://github.com/triton-inference-server/server/blob/main/CONTRIBUTING.md#coding-convention